typescript/infra/config/environments/mainnet3/warp/configGetters/getEverclearConfig.ts

```diff
@@ -0,0 +1,341 @@
+  base: '0x15a7cA97D1ed168fB34a4055CEFa2E2f9Bdb6C75',
+};
+
+// USDC token addresses extracted from signatures.json asset field
+const usdcTokenAddresses: Record<EverclearChain, string> = {
+  ethereum: '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48',
+  bsc: '0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d',
+  optimism: '0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85',
+  zksync: '0x1d17CBcF0D6D143135aE902365D2E5e2A16538D4',
+  arbitrum: '0xaf88d065e77c8cC2239327C5EDb3A432268e5831',
+  polygon: '0x3c499c542cEF5E3811e1192ce70d8cC03d5c3359',
+  avalanche: '0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E',
+  unichain: '0x078D782b760474a361dDA0AF3839290b0EF57AD6',
+  sonic: '0x29219dd400f2Bf60E5a23d13Be72B486D4038894',
+  mantle: '0x09Bc4E0D864854c6aFB6eB9A9cdF58aC190D0dF9',
+  ink: '0x2D270e6886d130D724215A266106e6832161EAEd',
+  linea: '0x176211869cA2b568f2A7D4EE941E073a821EE1ff',
+  scroll: '0x06eFdBFf2a14a7c8E15944D1F4A48F9F95F663A4',
+  zircuit: '0x3b952c8C9C44e8Fe201e2b26F6B2200203214cfF',
+  mode: '0xd988097fb8612cc24eeC14542bC03424c656005f',
+  base: '0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913',
+};
+
+// WETH token addresses extracted from signatures.json asset field
+const wethTokenAddresses: Record<EverclearChain, string> = {
+  ethereum: '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
+  bsc: '0x2170Ed0880ac9A755fd29B2688956BD959F933F8',
+  optimism: '0x4200000000000000000000000000000000000006',
+  zksync: '0x5AEa5775959fBC2557Cc8789bC1bf90A239D9a91',
+  arbitrum: '0x82aF49447D8a07e3bd95BD0d56f35241523fBab1',
+  polygon: '0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619',
+  avalanche: '0x49d5c2bdffac6ce2bfdb6640f4f80f226bc10bab',
+  unichain: '0x4200000000000000000000000000000000000006',
+  sonic: '0x50c42dEAcD8Fc9773493ED674b675bE577f2634b',
+  mantle: '0xdEAddEaDdeadDEadDEADDEAddEADDEAddead1111',
+  ink: '0x4200000000000000000000000000000000000006',
+  linea: '0xe5d7c2a44ffddf6b295a15c148167daaaf5cf34f',
+  scroll: '0x5300000000000000000000000000000000000004',
+  zircuit: '0x4200000000000000000000000000000000000006',
+  mode: '0x4200000000000000000000000000000000000006',
+  base: '0x4200000000000000000000000000000000000006',
+};
+
+// USDT token addresses extracted from signatures.json asset field
+const usdtTokenAddresses: Record<EverclearChain, string> = {
+  ethereum: '0xdAC17F958D2ee523a2206206994597C13D831ec7',
+  bsc: '0x55d398326f99059fF775485246999027B3197955',
+  optimism: '0x94b008aA00579c1307B0EF2c499aD98a8ce58e58',
+  zksync: '0x493257fD37EDB34451f62EDf8D2a0C418852bA4C',
+  arbitrum: '0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9',
+  polygon: '0xc2132d05d31c914a87c6611c10748aeb04b58e8f',
+  avalanche: '0x9702230A8Ea53601f5cD2dc00fDBc13d4dF4A8c7',
+  unichain: '0x588CE4F028D8e7B53B687865d6A67b3A54C75518',
+  sonic: '0x6047828dc181963ba44974801FF68e538dA5eaF9',
+  mantle: '0x201EBa5CC46D216Ce6DC03F6a759e8E766e956aE',
+  ink: '0x0000000000000000000000000000000000000000', // USDT address not available for ink
```

Comment:
I think that it would be better to not have this address set at all, as this might cause some unexpected behaviour of deploying a misconfigured warp route

Reply:
Done